modules/RTC/TraceablePeerConnection.js

```diff
@@ -2417,12 +2448,14 @@ TraceablePeerConnection.prototype.setRemoteDescription = function(description) {
             this.trace(
                 'setRemoteDescription::postTransform (sim receive)',
                 dumpSDP(description));
-
-            // eslint-disable-next-line no-param-reassign
-            description = this.tpcUtils.ensureCorrectOrderOfSsrcs(description);
         }
     }
 
+    if (this._usesUnifiedPlan) {
+        // eslint-disable-next-line no-param-reassign
+        description = this.tpcUtils.ensureCorrectOrderOfSsrcs(description);
```

Comment:
Is this operation ok when simulcast is off?

Reply:
Yes, this is needed even when simulcast is turned off as there will still be a FID group for the single source and the primary ssrc needs to be present before the secondary ssrc in the SDP.